gulpfile.js

```diff
@@ -0,0 +1,21 @@
+var gulp = require('gulp'),
+    connect = require('gulp-connect');
+
+gulp.task('connect', function() {
+  connect.server({
+    root: './src/',
```

Comment:
So I'd like your project to serve up the user list when you hit `http://localhost:1820/`. For this to work (in your projects current state), you'll need to serve `src/html/` and `src/css` separately by using the [multiple server](https://www.npmjs.com/package/gulp-connect#multiple-server) syntax. 

You'll also need to update the links in each of your html partials, but I assume you'll be able to figure that out.